R/set_endpoint.R

```diff
@@ -26,18 +26,15 @@
 set_endpoint = function(endpoint="production"){
 	
 	endpoint = tolower(endpoint)
+	match.arg(endpoint, c('production', 'prod', 'development', 'dev'))
```

Comment:
match.arg can do more than this code asks of it. i recommend this instead:
```r
set_endpoint = function(endpoint=c("production","development")){

  	endpoint = tolower(endpoint)
	endpoint <- match.arg(endpoint)

 	if(endpoint=="production"){
[etc.]
```

the above match.arg call should turn either 'prod' or 'production' (or 'pro', or 'p', or 'produc', etc.) into 'production'. same for development, all using partial matching. then the tests that follow only need to check for one value apiece. also, putting the options within the function declaration makes them part of the usage and, IMO, easier for users to find.

personally, i could do without line 28 altogether - i think it's reasonable to ask users to use lowercase based on the documented options available to them. but either way is OK with me.